task02/task02/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task02
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Радиус: ");
+
+            double r = Convert.ToDouble(Console.ReadLine());
```

Comment:
Методы класса Convert не стоит использовать без особых причин. Лучше так
```suggestion
            double r = double.Parse(Console.ReadLine());
```